AUTHORS

```diff
@@ -8,6 +8,7 @@ Contributors:
   Bastien Andrès, contributed to the dmx4linux plugin
   Carsten Presser, Karate USB plugin
   Dave Olsthoorn, working on a new web-ui
+  Florian Edelmann, added DMXCreator512Basic USB support
```

Comment:
Sorry, I probably didn't explain very well, feel free to stick to normal English when it's not a class name or whatever, so you can add some spaces back in here. It was more about being specific which model from DMX Creator it was, if someone wants to add the 1024 version.